core/storage/topic/gae_models.py

```diff
@@ -99,8 +99,8 @@ def get_export_policy(cls) -> Dict[str, base_models.EXPORT_POLICY]:
 class TopicModel(base_models.VersionedModel):
     """Model for storing Topics.
 
-    This class should only be imported by the topic services file
-    and the topic model test file.
+    This class should only be imported by the topic services file, the topic
+    fetchers file, and the topic model test file.
```

Comment:
Don't think you need to add topic fetchers -- see above.

Reply:
I reverted this change.